bin/ch/ch.cpp

```diff
@@ -129,8 +128,8 @@ void __stdcall PrintChakraCoreVersion()
                 // Doesn't matter if you are on 32 bit or 64 bit,
                 // DWORD is always 32 bits, so first two revision numbers
                 // come from dwFileVersionMS, last two come from dwFileVersionLS
-                printf("%s version %d.%d.%d.%d\n",
-                    chakraDllName,
+                wprintf(_u("%s version %d.%d.%d.%d\n"),
+                    GetChakraDllNameW(),
```

Comment:
For completeness, posting my standalone test:

```
$ cat test.cpp
#include <cstdio>
#include <wchar.h>
int main() {
    wprintf(L"%S", L"hello\n");
    return 0;
}

$ make test
g++     test.cpp   -o test

$ ./test
hello

$ sed -i test.cpp -e "s/%S/%s/"

$ make test
g++     test.cpp   -o test

$ ./test       # note: UTF-16 LE second byte is 0x00 NUL when wchar* interpreted as char*
h

$ vim test.cpp

$ cat test.cpp
#include <cstdio>
#include <wchar.h>
int main() {
    wprintf(L"%S", "hello\n"); // removed L prefix from arg 2, now ASCII string
    return 0;
}

$ make test
g++     test.cpp   -o test

$ ./test        # output is garbage because trying to interpret char* as wchar*
??%S
```